src/main/kotlin/io/snyk/plugin/services/SnykTaskQueueService.kt

```diff
@@ -74,16 +81,69 @@ class SnykTaskQueueService(val project: Project) {
 
                 if (settings.ossScanEnable) {
                     if (!getOssService(project).isCliInstalled()) downloadLatestRelease()
+                    waitForCliDownload()
```

Comment:
that will pause Code scan as well until CLI downloaded

Reply:
Yeah, probably better to change order.